.changeset/templates/svg-icons-release.md

```diff
@@ -0,0 +1,23 @@
+**BREAKING: Removed**
```

Comment:
This file will be described in the readme, but i figure i could add a "template" of release notes for icons.

Reply:
It's a manual process, not part of changeset